arkouda/pdarrayclass.py

```diff
@@ -1168,6 +1188,38 @@ def info(pda : Union[pdarray, str]) -> str:
         raise TypeError("info: must be pdarray or string".format(pda))
         return generic_msg(cmd="info", args="{}".format(pda))
 
+def list_registry() -> list[str]:
+    """
+    Return a list containing the names of all registered objects
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    list
+        List of all object names in the registry
+
+    Raises
+    ------
+    RuntimeError
+        Raised if there's a server-side error thrown
+    """
+    registered_list: list[str] = []
```

Comment:
I think the type string here should be `List[str]` vs `list[str]`
The python `typing` module uses the capitalized for for annotating typing info.